upgrade_test.py

```diff
@@ -118,6 +118,8 @@ def upgrade_node(self, node):
                 else:
                     node.remoter.run('sudo apt-get update')
                     node.remoter.run('sudo apt-get dist-upgrade {} -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" --force-yes --allow-unauthenticated'.format(scylla_pkg))
+        if self.db_cluster.params.get('test_sst3', default=None):
```

Comment:
1. why not `self.params`?
2. when introducing new params please add them to scylla.yaml with explanation what is their purpose

Reply:
Ok